frontend/.eslintrc.json

```diff
@@ -32,10 +32,12 @@
   "plugins": ["react", "prettier", "jest", "react-hooks"],
   "rules": {
     "indent": ["error", 2],
-    "linebreak-style": ["error", "unix"],
+    "linebreak-style": 0,
```

Comment:
this should be the form of `[level, (unix|windows)]`

Reply:
Yea was just getting some discrepancies from my Mac and Windows environments. Currently off but I'll reintroduce this solution in the next PR